spec/support/mocks_and_stubbs/openlibrary_mock.rb

```diff
@@ -0,0 +1,38 @@
+module OpenlibraryMock
+  BASE_URI = 'https://openlibrary.org/api'.freeze
+
+  def successfuly_book_get(isbn)
+    stub_request(:get, "#{BASE_URI}/books")
+      .with(
+        query: { bibkeys: "ISBN:#{isbn}", format: 'json', jscmd: 'data' }
+      )
+      .to_return(
+        status: 200,
+        headers: { 'Content-Type' => 'application/json' },
+        body: File.read('./spec/support/fixtures/openlibrary/response_success.json')
+      )
+  end
+
+  def empty_book_get(isbn)
+    stub_request(:get, "#{BASE_URI}/books")
+      .with(
+        query: { bibkeys: "ISBN:#{isbn}", format: 'json', jscmd: 'data' }
+      )
+      .to_return(
+        status: 200,
+        headers: { 'Content-Type' => 'application/json' },
+        body: '{}'
+      )
+  end
+
+  def internal_server_error_book_get(isbn)
```

Comment:
:warning: Warning: this not be an internal error. This is an openlibrary error
```suggestion
  def openlibrary_server_error_book_get(isbn)
```